PhysicsTools/PatAlgos/plugins/PATElectronProducer.cc

```diff
@@ -323,6 +336,7 @@ void PATElectronProducer::produce(edm::Event & iEvent, const edm::EventSetup & i
 	  const edm::RefToBase<reco::GsfElectron>& elecsRef = electrons->refAt(idx);
 	  Electron anElectron(elecsRef);
 	  anElectron.setPFCandidateRef( pfRef  );
+    anElectron.setIsolationPUPPI((*PUPPIIsolation_charged_hadrons)[elePtr], (*PUPPIIsolation_neutral_hadrons)[elePtr], (*PUPPIIsolation_photons)[elePtr]);
```

Comment:
The same implementation needs to be done for not "useParticleFlow_". 
Under this condition, only electrons identified as electron in Particle flow algorithm will have the puppi isolation. 
But we would like to have the puppi isolation for all gedElectrons.